### DISCUSSION TITLE
Woche 8.01-14.01.24

### DISCUSSION THREAD

**Discussion by carolina-cza** [carolina-cza/Tecna#6] [Weekly Blog]
Halli hallo. Die Woche war bisschen inaktiver und unkreativer, aber so Wochen/Tagen gibt es natürlich auch mal. Aber die Woche habe ich mir ein Tutorial angeschaut wie ein einfacher Chatboot funktioniert und in etwa aufgebaut [ist().](https://github.com/carolina-cza/Tecna/blob/codespace-super-space-yodel-x55ppqjx97jqfp699/daily_stuff/chatbot.py) Ich bin mit dem ergebniss des Tutorials nicht ganz so zufrieden deswegen werde ich die nächsten Wochen mal schauen ob man das noch bisschen praktischer gestallten kann. Im weiteren hab ich mir paar nested loops angeschaut und dazu paar tutorials gemacht, einfach nur für die Übung und das bessere Verständnis.
Und joa das war es eigentlich auch schon was ich die Woche so programmiert hab.

**Comment by Amalia-Dann** [comment]
Sehr gut, dass du reflektierst, wie die Woche war. 
Hoffentlich klappt das mit dem Chatbot in der Zukunft :)

**Comment by DGrillCoder** [comment]
So Wochen/ Tage gibt es :D
Finde es toll, dass du trotzdem dran bleibst und was machst. Chatbot hört sich mega spannend an.
Vl. sehen wir ja mal ein Tutorial von dir über Chatbots
